run/system-package/main.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, make_response, request
+import os
+import subprocess
+import sys
+
+app = Flask(__name__)
+
+# [START run_system_package_handler]
+@app.route('/diagram.png', methods=['GET'])
+def index():
```

Comment:
A comment as to what this method does would be nice.

For example, we seem to expect a `dot` query parameter in our URL. What is the shape of that query param? What is an example? Is there a link to what this type of data structure?

Reply:
Added a comment - is it clearer?

Reply:
I still don't understand what this method does by reading the comment.

```
Uses the graphviz DOT language
https://www.graphviz.org/doc/info/lang.html
```

From reading the code, it seems like this method takes a HTTP GET request with query param `dot` and returns a png with the rendered DOT diagram in a HTTP response.

I think it would be clearer if the comment stated something more along that.

Reply:
Updated.